test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -701,7 +701,7 @@ public void GetDatabaseWithQueryConnectionTest()
         /// Get Definition for a object by putting the cursor on 3 different
         /// objects
         /// </summary>
-        [Fact]
+       // [Fact]

```

Comment:
please add // todo comment 

---
In reply to: [114197960](https://github.com/Microsoft/sqltoolsservice/pull/332#discussion_r114197960) [](ancestors = 114197960)